2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -403,3 +403,54 @@ Il piano di progetto è il documento principale che viene redatto durante questo
 + Consuntivo del progetto.
 
 La redazione del documento va di pari passo con l'avanzamento del progetto, in modo tale da essere sempre aggiornato alla situazione corrente del lavoro.
+
+== Processo di valutazione e controllo di progetto <valutazioneControllo>
+_Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.3.2_
+=== Scopo
+Il processo di valutazione e controllo ha lo scopo di determinare lo stato del progetto tramite la verifica dell'allineamento del lavoro con i piani definiti. Tale allineamento viene dimostrato osservando i seguenti parametri:
+
+- tempistiche;
+- costi;
+- obiettivi raggiunti.
+
+Questo viene fatto all'interno di revisioni periodiche chiamate retrospettive, e viene messo per iscritto all'interno dei verbali di tali riunioni e nei paragrafi di consuntivo del Piano di progetto.
+
+=== Attività
+==== Piano di valutazione e controllo del progetto
+La valutazione e il controllo del progetto vengono fatte settimanalmente, in corrispondenza della riunione di retrospettiva. Durante questa riunione si valuta ciò che è stato svolto durante il periodo di lavoro (_sprint_) che si sta concludendo, se ne identificano i problemi e si decidono soluzioni di controllo che vadano a risolvere o arginare i problemi individuati.
+
+Oltre a queste revisioni periodiche interne, sono presenti attività nelle quali il gruppo espone la propria situazione al Proponente, tramite riunione su Zoom, e Committente, tramite le attività denominate "diario di bordo".
+
+Inoltre, è necessario che il gruppo svolga delle revisioni tecniche con il Committente per avere una valutazione esterna. Queste due revisioni sono chiamate Requirements and Technology Baseline (o RTB) e Product Baseline (o PB).
+
+==== Valutazione <valutazione>
+Durante l'attività di valutazione il gruppo deve analizzare la situazione del progetto, e per fare ciò deve adottare degli strumenti che rappresentino tale situazione nel modo più oggettivo possibile. Il momento in cui si effettua questo compito è la retrospettiva settimanale. Questa viene fatta tramite meeting interno su Discord con il supporto di una board Miro. Il meeting è suddiviso concettualmente in tre parti, rappresentate all'interno della board da tre riquadri:
+
+- Keep doing: raccoglie tutti gli aspetti positivi (e di conseguenza da mantenere) dello sprint;
+- Improvements: raccoglie tutte le problematiche incontrate durante lo sprint;
+- Task: raccoglie tutte le attività da svolgere nel prossimo sprint, alcune delle quali direttamente collegate agli elementi appartenenti ad "Improvements".
+
+Tramite Miro il gruppo riesce ad avere una panoramica della situazione del lavoro, che viene poi completata da Jira, l'Issue Tracking Systema adottato. Al suo interno il gruppo ha definito le due milestone principali (RTB e PB), nelle quali ha creato delle Epic che rappresentano ciò che è necessario produrre per quella milestone, e che raccolgo tutti i task necessari alla produzione di tali prodotti (documenti o software).
```

Comment:
oltre che principali, definirei le milestone anche "esterne", in quanto internamente ci fissiamo un numero decisamente più elevato di milestone anch'esse principali.